packages/@rnw-scripts/babel-node-config/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@rnw-scripts/babel-node-config",
+  "version": "0.0.0",
+  "license": "MIT",
+  "main": "babel.config.js",
+  "devDependencies": {
+    "@babel/core": "^7.8.4",
+    "@babel/preset-env": "^7.8.4",
+    "@babel/preset-typescript": "^7.8.3",
+    "eslint": "6.8.0"
+  },
+  "peerDependencies": {
+    "@babel/core": "^7.8.0",
```

Comment:
So, this actually generates a whole bunch of yarn warnings because yarn thinks we don't fulfill these (since the dependent of the package adds them), but it's the more correct thing to do.

Reply:
Perhaps I’m missing something, but if you match these version requirements above in `devDependencies` I would not expect to see any warnings.

Reply:
Yeah, I've also been confused by this. We see the same issue for other packages. Wondering if there's a Yarn bug with workspaces or something.

Reply:
Yeah, I think yarn doesn't see devDeps as fulfilling local peer deps... seems like a yarn bug to me.